packages/office-ui-fabric-react/src/components/Persona/Persona.tsx

```diff
@@ -98,12 +97,13 @@ export class Persona extends BaseComponent<IPersonaProps, IPersonaState> {
       onRenderOptionalText,
       imageShouldStartVisible
     } = this.props;
+    let size = this.props.size as PersonaSize;
```

Comment:
Should variable creation be using static typing  (e.g. ':' type assignment) as opposed to 'as' casting?